sympy/solvers/bivariate.py

```diff
@@ -168,17 +168,16 @@ def _lambert(eq, x):
 
     # solution of the given Lambert equation is like
     # sol = -c/b + (a/d)*LambertW(arg, k),
-    # where arg = d/(a*b)*exp((c*d-b*f)/a/b) and k in lambert_real_branches
-    # arg can have more than one values as exp(num/p*den)[exp((c*d-b*f)/a/b)]
+    # where arg = d/(a*b)*exp((c*d-b*f)/a/b) and k in lambert_real_branches.
+    # arg can have more than one values as exp(num/den)**(1/p)[exp((c*d-b*f)/a/b)]
     # will required to have all solutions of x**p - exp(num/den).
 
     # calculating args for LambertW
     num, den = ((c*d-b*f)/a/b).as_numer_denom()
     p, den = den.as_coeff_Mul()
-    args = [d/(a*b)*exp(num/den)]
-    if p != S.One:
-        exp_t = Dummy('exp_t')
-        args = [d/(a*b)*e for e in solve(exp_t**p - exp(num/den), exp_t)]
+    exp_t = Dummy('exp_t')
+    args = [d/(a*b)*e if p != S.One else d/(a*b)*exp(num/den) \
```

Comment:
```python
e = exp(num/den)
args = solve(exp_t**p - e, exp_t) if p is S.One else [d/(a*b)*e]
```
Should this just be `args = roots(expt**p - e, exp_t).keys()`?

Reply:
What does the variable `exp_t`  mean? (When I used `xe` it was representing the roots of `e`.)

Reply:
from exp_t I meant to be roots of exponential form and t to don't confuse it with the symbol x because most people refer the variable in the main equation as x. 
> args = roots(exp_t**p - e, exp_t).keys()

I didn't know this modification before. I will try this and check it out. Thanks.